src/pages/home/report/ReportActionItemCreated.js

```diff
@@ -38,28 +41,46 @@ const defaultProps = {
 const ReportActionItemCreated = (props) => {
     const isPolicyExpenseChat = ReportUtils.isPolicyExpenseChat(props.report);
     const icons = ReportUtils.getIcons(props.report, props.personalDetails, props.policies);
+
+    // The image handling depends on how large the chat window is
+    const backgroundImageHeight = Dimensions.get('window').height < Dimensions.get('window').width
+        ? 450
+        : 300;
+
     return (
         <OfflineWithFeedback
             pendingAction={lodashGet(props.report, 'pendingFields.addWorkspaceRoom') || lodashGet(props.report, 'pendingFields.createChat')}
             errors={lodashGet(props.report, 'errorFields.addWorkspaceRoom') || lodashGet(props.report, 'errorFields.createChat')}
             errorRowStyles={styles.addWorkspaceRoomErrorRow}
             onClose={() => Report.navigateToConciergeChatAndDeleteReport(props.report.reportID)}
         >
+            <View style={[styles.emptyStateBackgroundContainer, {
+                // since height and top are dynamically set they needs to be updated here, not in styles.js
+                height: backgroundImageHeight,
+                top: -backgroundImageHeight,
+            }]}
+            >
```

Comment:
When you say "dynamically set" do you mean as the user changes window size?

Reply:
I guess `dynamically` is wrong. It's only set when the report first loads. If there's an easier way to tell between landscape and portrait I'm all ears.

Reply:
Okay in terms of cleanup- one idea is moving all of this (including the backgroundImageHeight variable declaration) into StyleUtils and then calling StyleUtils.get_____() to return the correct style object

Reply:
oh neat, I didn't know `StyleUtils` existed! That's a great idea